core/src/test/scala/aima/core/environment/vacuum/ModelBasedReflexVacuumAgentSpec.scala

```diff
@@ -24,9 +24,6 @@ class ModelBasedReflexVacuumAgentSpec extends Specification {
   "should assume dirty after moving to location B" in new context {
     agent.agentFunction.apply(LocationAPercept)
     agent.agentFunction.apply(CleanPercept)
-    agent.agentFunction.apply(NoPercept) must beLike {
```

Comment:
looks like  you have an example without an assertion now
this seems like a valid example